core/tests/dispatch_integration_tests.rs

```diff
@@ -600,85 +613,73 @@ fn remote_delivery_to_registered_actors_lazy_mixed_udp() {
 #[ignore]
 fn remote_lost_and_continued_connection() {
     let mut net_cfg = NetworkConfig::default();
-    net_cfg.set_max_connection_retry_attempts(6);
-    net_cfg.set_connection_retry_interval(500);
-    let system = system_from_network_config(net_cfg);
-    let remote_a = system_from_network_config(NetworkConfig::default());
-    let remote_port = remote_a.system_path().port();
-
-    //let (ponger_unique, pouf) = remote.create_and_register(PongerAct::new);
-    let (ponger_named, ponf) = remote_a.create_and_register(PongerAct::new_lazy);
-    let poaf = remote_a.register_by_alias(&ponger_named, "custom_name");
-    ponf.wait_expect(Duration::from_millis(1000), "Ponger failed to register!");
-    poaf.wait_expect(Duration::from_millis(1000), "Ponger failed to register!");
+    let ping_interval = Duration::from_millis(100);
+
+    net_cfg.set_max_connection_retry_attempts(CONNECTION_RETRY_ATTEMPTS);
+    net_cfg.set_connection_retry_interval(CONNECTION_RETRY_INTERVAL);
+    let pinger_system = system_from_network_config(net_cfg);
+    let ponger_system_1 = system_from_network_config(NetworkConfig::default());
+    let ponger_system_port = ponger_system_1.system_path().port();
+
+    let (ponger_named, _) = start_ponger(&ponger_system_1, PongerAct::new_lazy());
+    ponger_system_1
+        .register_by_alias(&ponger_named, "custom_name")
+        .wait_expect(REGISTRATION_TIMEOUT, "Ponger failed to register!");
     let named_path = ActorPath::Named(NamedPath::with_system(
-        remote_a.system_path(),
+        ponger_system_1.system_path(),
         vec!["custom_name".into()],
     ));
-    let named_path_clone = named_path.clone();
 
-    let (pinger_named, pinf) =
-        system.create_and_register(move || PingerAct::new_lazy(named_path_clone));
-    pinf.wait_expect(Duration::from_millis(1000), "Pinger failed to register!");
+    let (pinger, pif) =
+        pinger_system.create_and_register(move || PingStream::new(named_path, ping_interval));
+    pif.wait_expect(REGISTRATION_TIMEOUT, "Pinger failed to register!");
 
-    remote_a.start(&ponger_named);
-    system.start(&pinger_named);
+    ponger_system_1.start(&ponger_named);
+    pinger_system.start(&pinger);
 
-    // Wait for the pingpong
-    thread::sleep(Duration::from_millis(2000));
+    thread::sleep(WAIT_TIME); // ping-pong for a while...
```

Comment:
Instead of doing it like this, can't we wait for a fixed number of pongs? Like, clearly we can loose some, so we don't count how many we sent, but we count how many we received and if we received X, then we go: "Ok, so clearly probabilistically a bunch of them are coming through, so there's no infinite loss. Good enough for UDP."